FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/FirebaseAuthSwiftUI/Views/FUIAuthPicker.swift

```diff
@@ -28,18 +28,23 @@ public protocol AuthPickerViewProtocol: View {
   var configuration: AuthPickerViewConfiguration { get }
 }
 
-public struct AuthPickerView: AuthPickerViewProtocol {
+public struct AuthPickerView<Content: View>: AuthPickerViewProtocol {
   public let configuration: AuthPickerViewConfiguration
+  private let content: Content
 
-  public init(configuration: AuthPickerViewConfiguration = AuthPickerViewConfiguration()) {
+  public init(configuration: AuthPickerViewConfiguration = AuthPickerViewConfiguration(),
```

Comment:
The configuration would probably be either a view modifier, or better yet, a style.